src/Traits/Debouncable.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Mpbarlow\LaravelQueueDebouncer\Traits;
+
+use Mpbarlow\LaravelQueueDebouncer\Facade\Debouncer;
+
+trait Debouncable
+{
+	/**
+	 * Dispatch the job with the given arguments.
+	 *
+	 * @return \Illuminate\Foundation\Bus\PendingDispatch
+	 */
+	public static function debounce()
+	{
+		$jobProperties = func_get_args();
```

Comment:
```suggestion
	public static function debounce(...$jobProperties)
	{
```
I would prefer a variadic argument here, to help prevent IDEs complaining about passing arguments to functions that don't claim to accept any.

Reply:
Not a problem - i was mimicking the code from the dispatchable trait within laravel itself. This way is cleaner, i like it.